README.md

```diff
@@ -2,48 +2,149 @@
+* Fixed the bug with year and month getting trimmed for file system images.
+
+### 2.0 ###
+* Updated to use Microsoft Azure SDK for PHP from https://github.com/WindowsAzure/azure-sdk-for-php and fixed to work with WordPress 3.4.1
+
+### 1.9 ###
+* Fixed case sensitivity error in file names on Linux
+
+### 1.8 ###
+* Bug fixed in generating blob storage URL when using Microsoft Azure Storage emulator
+
+### 1.7 ###
+* Added support to upload video files to blob storage
+
+### 1.5 ###
+* Included Microsoft Azure SDK for PHP v4.1.0 with the plugin. Now setting mime-type for uploaded file to blob storage.
+
+### 1.4 ###
+* Included Microsoft Azure SDK for PHP v4.0.2 with the plugin.
+
+### 1.3 ###
+* Included Microsoft Azure SDK for PHP v4.0.1 with the plugin, so no need to install the SDK separetely. Also fixed thumbnail handling issue while uploading files when some specific theme is enabled.
+
+### 1.2 ###
+* This release is compatible with Microsoft Azure SDK for PHP v3.0.0. It also fixes issue with deleting media files when thumbnails are associated.
+
+### 1.1 ###
+* This release is compatible with Microsoft Azure SDK for PHP v2.1.0 and WordPress 3.1
+
+### 1.0 ###
+* First release of Microsoft Azure Storage plugin for WordPress
+
+
+## Upgrade Notice ##
+
+### 3.0.0 ###
 This release features several security fixes and enhancements.
 It is highly recommended that all users upgrade immediately.
 
-## Support Level
-**Stable:** 10up is not planning to develop any new features for this, but will still respond to bug reports and security concerns.  We welcome PRs, but any that include new features should be small and easy to integrate and should not include breaking changes.  We otherwise intend to keep this tested up to the most recent version of WordPress.
-
-## Known Issues
 
-### Storage Account Versions
-Storage accounts can be created via CLI, classic Azure portal, or the new Azure portal, with varying results.
+## Known Issues ##
 
-If a Storage account is created with the new Azure portal, authentication will fail, resulting in the inability to view/add containers or files. Creating a Storage account with the Azure CLI should allow the plugin to work with new Storage accounts.
+### Storage Account Versions ###
+Storage accounts can be created via CLI, classic Azure portal, or the new Azure portal,
+with varying results.
 
-### Responsive Images in WordPress 4.4
-Images uploaded to the Azure Storage service will not automatically receive responsive versions. Images added through the WordPress Media Loader *should* get automatically converted to responsive images when inserted into a post or page. We are investigating options for full support of responsive images in the plugin.
+If a Storage account is created with the new Azure portal, authentication will fail,
+resulting in the inability to view/add containers or files. Creating a Storage account
+with the Azure CLI should allow the plugin to work with new Storage accounts.
```

Comment:
The replaced content here appears more accurate for these issues, unless I'm reading them wrong?